### PR TITLE
Fix off by one error in header helper

### LINKED ISSUE #3355: make_fitswcs_header has an off by one error in the default crpix calculation
This needs double checking but it looks like we  are +1 off in the default array shape halving

### PR DESCRIPTION
**Pull request by Cadair**
Fixes #3355

### CODE REVIEW

**Comment by astrofrog** [comment]
Seems reasonable to me!

**Comment by Cadair** [comment]
Online test fails are unrelated.

File: changelog/3356.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Fix off by one error in `sunpy.map.make_fiteswcs_header` where when using the
```
Comment (wtbarnes) [inline changelog/3356.bugfix.rst]: `sunpy.map.make_fiteswcs_header` should be `sunpy.map.make_fitswcs_header`

Comment (Cadair) [reply changelog/3356.bugfix.rst]:
```suggestion
Fix off by one error in `sunpy.map.make_fitswcs_header` where when using the
```

**Pull request merged**